scanner/utils.go

```diff
@@ -13,35 +14,29 @@ import (
 )
 
 func askForOptionValue(option models.OptionNode) (string, string, error) {
```

Comment:
pls create a separate task to support the new input types in our interactive tools, including the scanner itself